pkg/pillar/hypervisor/kvm.go

```diff
@@ -123,6 +126,21 @@ const qemuConfTemplate = `# This file is automatically generated by domainmgr
   driver = "intel-iommu"
   caching-mode = "on"
 {{ end }}
+
+{{if eq .VirtualizationMode "FML"}}
+# UEFI boot using OVMF
+[drive "drive-ovmf-code"]
+  if = "pflash"
+  format = "raw"
+  readonly = "on"
+  file = "{{.DomainConfig.BootLoader}}"
+
+[drive "drive-ovmf-vars"]
+  if = "pflash"
+  format = "raw"
```

Comment:
maybe it makes sense to specify "readonly" "off"

Reply:
Maybe... This I will understand during the test)))

Reply:
I've grepped it over the internet, usually code is read-only, vars is not.

Reply:
Yeah, I also saw it. I'm just not sure we have set the read-only option to "off" explicitly.

Reply:
in my test, at least ubuntu boots up fine with having `readonly` for the OVMF_VAR.fd too.